test/Driver/android-link.swift

```diff
@@ -1,10 +1,10 @@
 // RUN: %empty-directory(%t)
-// RUN: %swift_driver_plain --driver-mode=swiftc -target armv7-none-linux-androideabihf -emit-executable %s -### 2>&1 | %FileCheck %s -check-prefix CHECK-ARMv7
-// CHECK-ARMv7: clang{{(.exe)?"?}} -target armv7-unknown-linux-androideabi
+// RUN: %swift_driver_plain --driver-mode=swiftc -target armv7-unknown-linux-androideabihf -emit-executable %s -### 2>&1 | %FileCheck %s -check-prefix CHECK-ARMv7
+// CHECK-ARMv7: clang{{(.exe)?"?}} -target armv7-unknown-linux-androidhf

```

Comment:
The failing test was that I kept all the `eabi` strings in the clang command output before, but [the LLVM triple normalizes them out](https://github.com/apple/llvm-project/blob/f77c948d56b09b839262e258af5c6ad701e5b168/llvm/lib/Support/Triple.cpp#L960).